nova_vm/src/ecmascript/types/spec/data_block.rs

```diff
@@ -57,9 +57,15 @@ impl Drop for DataBlock {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+pub(crate) struct U8Clamped(pub u8);
```

Comment:
**praise:** This is the good and proper solution to U8Clamed, I think.

**nitpick:** Probably unnecessary, but this could be `#[transparent]`.